### PR TITLE
Project approval status

### LINKED ISSUE #1516: Have the status of the site whether approved or not listed under visability
Currently you can't tell whether your project is approved or not by the Zooniverse in the project builder. The only way you know is if you see it on the Zooniverse projects list and the line on the bottom saying 'This project has been built using the Zooniverse Project Builder but is not yet an official Zooniverse project. Queries and issues relating to this project directed at the Zooniverse Team may not receive any response.' disappears. 

I think, It would be good to show whether the request was approved with some kind of message on the 'Project state and visibility' - something like 'Zooniverse approved'?

screen shot 2015-09-03 at 8 56 19 am

### PR DESCRIPTION
**Pull request by saschaishikawa**
Add feature to display current approval status of a project (resolves #1516)

### CODE REVIEW

**Comment by chrissnyder** [comment]
Rebase and clean up git history please.

File: app/pages/lab/visibility.cjsx [hunk 1]
```diff
@@ -19,6 +19,8 @@ module.exports = React.createClass
   setterProperty: 'project'
 
   render: ->
+    betaApprovalLabel = null
+    launchApprovalLabel = null
```
Comment (chrissnyder) [inline app/pages/lab/visibility.cjsx]: These don't look to be used?

**Comment by chrissnyder** [comment]
Bit of a nitpick on style, but can you your formatting on the conditionals to something like https://github.com/zooniverse/Panoptes-Front-End/blob/master/app/collections/home.cjsx#L18-L26 to match the rest of the app?

Also, can you add what issues are being closed to the PR description? I think they were lost when the commits were squashed.

Other than that, looks good!

**Comment by saschaishikawa** [comment]
Should be done now.

**Comment by chrissnyder** [comment]
LGTM. Last request, can you add what issue is being closed here to the PR description?

**Comment by saschaishikawa** [comment]
Done

**Pull request merged**